.github/workflows/playwright.yml

```
@@ -75,11 +82,29 @@ jobs:
-          name: allure-report
-          path: ./downloaded-allure-report
+          path: ./allure-results
+
+      # Объединяем
+      - name: Merge Allure Results
+        run: |
+          mkdir merged-results
+          find allure-results -mindepth 2 -maxdepth 2 -type d -name "allure-results" -exec cp -r {}/* merged-results/ \;
+
+      - uses: pnpm/action-setup@v4
+        version: 8
```

actionlint (1.7.12)

[error] unexpected key "version" for step to execute action. expected one of "continue-on-error", "env", "id", "if", "name", "timeout-minutes", "uses", "with"

(syntax-check)